lib/input/azure_queue_storage.go

```diff
@@ -91,7 +99,7 @@ func (a *azureQueueStorage) ReadWithContext(ctx context.Context) (msg types.Mess
 			part := message.NewPart([]byte(queueMsg.Text))
 			meta := part.Metadata()
 			meta.Set("queue_storage_insertion_time", queueMsg.InsertionTime.Format(time.RFC3339))
-			meta.Set("queue_storage_queue_name", a.conf.QueueName)
+			meta.Set("queue_storage_queue_name", a.queueName.String(0, msg))
```

Comment:
The value here might be different if the interpolation contains time based or random functions, we should capture the result of the first execution and use it in both cases.

Reply:
You're right, changed